minikube/startup.sh

```diff
@@ -22,9 +22,10 @@ kubectl config set-context minikube --namespace=psga-minikube
 echo "Setting service account"
 kubectl apply -f service_account.yaml
 
-kubectl create secret generic regcred \
-  --from-file=.dockerconfigjson=$HOME/.docker/config.json \
-  --type=kubernetes.io/dockerconfigjson
+# this raises an error if already present
+#kubectl create secret generic regcred \
+#  --from-file=.dockerconfigjson=$HOME/.docker/config.json \
+#  --type=kubernetes.io/dockerconfigjson
```

Comment:
I've commented this only because it raises an error if it is already present.
Is it possible to `apply` instead of `create`?